tuner/tuner/merge_td_specs_test.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2024 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""
+Usage: python -m pytest merge_td_specs_test.py
+"""
+
+import pytest
+
+from typing import Generator
+
+from iree.compiler import ir  # type: ignore
+
+from . import merge_td_specs
+from . import common
+
+
+@pytest.fixture
+def tuner_ctx() -> Generator[common.TunerContext, None, None]:
+    from logging import Logger
+    from unittest.mock import MagicMock
+
+    mock_logger = MagicMock(spec=Logger)
+    with common.TunerContext(logger=mock_logger) as ctx:
+        yield ctx
```

Comment:
Can we put this and similar functions in a new file with common testing utilities? Say, `test_utils.py`.